analysers/analyser_merge_radio_support_FR.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Jérôme Amagat 2019                                         ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+from io import open
+
+class _Analyser_Merge_Radio_Support_FR(Analyser_Merge):
+    def __init__(self, config, logger, clas, NAT_IDs, title, tags_select, tags_generate):
+
+        self.missing_official = {"item":"8370", "class": 1+10*clas, "level": 3, "tag": ["merge"], "desc": T_f(u"Radio support (%s) not integrated" % title) }
+        self.possible_merge   = {"item":"8371", "class": 3+10*clas, "level": 3, "tag": ["merge"], "desc": T_f(u"Radio support (%s), integration suggestion" % title) }
+        self.update_official  = {"item":"8372", "class": 4+10*clas, "level": 3, "tag": ["merge"], "desc": T_f(u"Radio support (%s) update" % title) }
+
+        self.communeNameIndexedByInsee = {}
+        with open("dictionaries/FR/BddCommunes", "r", encoding="utf-8") as f :
+            for x in f :
+                x = x.split("\t")
+                code_insee = x[0]
+                name_insee = x[1].strip()
+                self.communeNameIndexedByInsee[code_insee] = name_insee
+                
+        Analyser_Merge.__init__(self, config, logger,
+            u"https://www.data.gouv.fr/datasets/551d4ff3c751df55da0cd89f",
```

Comment:
Please, use the long url https://www.data.gouv.fr/fr/datasets/donnees-sur-les-installations-radioelectriques-de-plus-de-5-watts-1/

Reply:
modified